[2022-05-16 12:55:43.225] [remoteagent] [info] Extension host agent started.
[2022-05-16 12:55:43.519] [remoteagent] [info] [127.0.0.1][4460b690][ManagementConnection] New connection established.
[2022-05-16 12:55:43.917] [remoteagent] [info] [127.0.0.1][585856a3][ExtensionHostConnection] New connection established.
[2022-05-16 12:55:43.984] [remoteagent] [info] [127.0.0.1][585856a3][ExtensionHostConnection] <109> Launched Extension Host Process.
[2022-05-16 12:57:18.946] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: terminated by itself with code null, signal: SIGSEGV
[2022-05-16 12:58:58.545] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: terminated by itself with code null, signal: SIGSEGV
[2022-05-16 12:58:58.727] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: inotify_add_watch on '/home/wild/.npm/_npx/809/lib/node_modules/.staging' failed: No such file or directory (EUNKNOWN) (path: /home/wild)
[2022-05-16 12:58:58.728] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: inotify_add_watch on '/home/wild/.npm/_npx/809/lib/node_modules/.staging' failed: No such file or directory
[2022-05-16 13:00:13.675] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: terminated by itself with code null, signal: SIGSEGV
[2022-05-16 13:00:13.876] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: inotify_add_watch on '/home/wild/.npm/_npx/1294/lib/node_modules/.staging' failed: No such file or directory (EUNKNOWN) (path: /home/wild)
[2022-05-16 13:00:13.877] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: inotify_add_watch on '/home/wild/.npm/_npx/1294/lib/node_modules/.staging' failed: No such file or directory
[2022-05-16 13:14:47.104] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/wild)
[2022-05-16 13:14:47.104] [remoteagent] [error] [File Watcher (universal)] gave up attempting to restart watcher after error: std::bad_alloc
[2022-05-16 13:14:47.920] [remoteagent] [warning] [File Watcher (node.js)] Watcher shutdown because watched path got deleted
[2022-05-16 13:56:08.026] [remoteagent] [info] [127.0.0.1][585856a3][ExtensionHostConnection] <109> [0mGET /favicon.ico [33m404[0m 6.007 ms - 150[0m

[2022-05-16 14:53:56.221] [remoteagent] [info] [127.0.0.1][4460b690][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-05-16 14:53:56.294] [remoteagent] [info] [127.0.0.1][585856a3][ExtensionHostConnection] <109> Extension Host Process exited with code: 0, signal: null.
